File: solutions/beecrowd/1011/1011.js
const readline = require('readline')

const PI = 3.14159

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
})

rl.on('line', function (line) {
  console.log(`VOLUME = ${(4.0 / 3.0 * PI * Math.pow(parseInt(line), 3)).toFixed(3)}`)
})

File: solutions/beecrowd/1011/1011.cs
using System;

class Solution {
    static void Main() {
        double r, pi = 3.14159;
        string line;

        while ((line = Console.ReadLine()) != null) {
            r = Convert.ToDouble(line);
            Console.WriteLine("VOLUME = {0:F3}", 4.0 / 3.0 * pi * Math.Pow(r, 3));
        }
    }
}
